Source/Core/Common/Logging/LogManager.h

```diff
@@ -66,15 +75,18 @@ class LogContainer
 
 	void SetLevel(LogTypes::LOG_LEVELS level) { m_level = level; }
 
-	bool HasListeners() const { return !m_listeners.empty(); }
+	bool HasListeners() const { return bool(m_listener_ids); }
+
+	typedef class BitSet32::Iterator iterator;
```

Comment:
I believe you can use a type alias (using iterator = BitSet32::Iterator) to get rid of the class, but that might only apply to templates, I forget.